apps/grader-host/tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "extends": "@prairielearn/tsconfig/tsconfig.app.json",
   "compilerOptions": {
     "rootDir": "./src",
-    "outDir": "./dist"
+    "outDir": "./dist",
```

Comment:
Is this intentional or is prettier messing up JSON files?

Reply:
`tsconfig.json` is treated as JSONC, so this is not "messing up".